.gitignore

```diff
@@ -1 +1,2 @@
 /target
+*.db*
```

Comment:
You don't end your sqlite dbs with `sqlite`? hmmm I've been doing it wrong lol

Reply:
Yeah I was a bit confused by this myself. The `sqlx` docs used `.db`, so I went with it, but maybe better to use `sqlite` extensions.

Reply:
I'll get used to using `.db` until we (inevitably?) move to postgres